views/Navbar.tsx

```diff
@@ -1,159 +1,260 @@
-import React, { useState, useEffect } from 'react';
-import { useSession } from 'next-auth/react'
-import type { Session } from 'next-auth'
-import Link from 'next/link';
-import { AiOutlineMenu, AiOutlineClose } from 'react-icons/ai';
+import React, { useState, useEffect } from "react";
+import { useSession } from "next-auth/react";
+import type { Session } from "next-auth";
+import Link from "next/link";
+import { AiOutlineMenu, AiOutlineClose } from "react-icons/ai";
 import LoginLogout from "../components/LoginLogout";
-import chromeLogo from '../public/chrome-logo.png'
-import Image from 'next/image';
-import RudderContext from '../components/RudderContext';
-import { getAndSetAnonymousIdFromLocalStorage } from '../utils/rudderstack_initialize';
-import { getAuthUserId, getAuthUserName } from '../utils/auth';
-import AppBar from '../components/AppBar';
-import VibinexDarkLogo from '../public/vibinex-dark-logo.png';
-import Button from '../components/Button';
+import chromeLogo from "../public/chrome-logo.png";
+import Image from "next/image";
+import RudderContext from "../components/RudderContext";
+import { getAndSetAnonymousIdFromLocalStorage } from "../utils/rudderstack_initialize";
+import { getAuthUserId, getAuthUserName } from "../utils/auth";
+import AppBar from "../components/AppBar";
+import VibinexDarkLogo from "../public/vibinex-dark-logo.png";
+import Button from "../components/Button";
```

Comment:
Please remove any inconsequential changes from your pull request. 

When adding your changes using git, use the `-p` flag to choose which changes you want to commit and which ones you want to skip. These changes should have been skipped.